hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/DiskBalancerManager.java

```diff
@@ -59,6 +59,8 @@ public class DiskBalancerManager {
   private final NodeManager nodeManager;
   private Map<DatanodeDetails, DiskBalancerStatus> statusMap;
   private Map<DatanodeDetails, Long> balancedBytesMap;
+  private long successMoveCount;
```

Comment:
Move successMoveCount and failureMoveCount into DiskBalancerStatus. 

successMoveCount and failureMoveCount are per datanode, not the sum of all datanodes.